Update desktop-cross-report-drill-through.md (#2184)
MicrosoftDocs/powerbi-docs: powerbi-docs/create-reports/desktop-cross-report-drill-through.md

This article shows you how to set up and use cross-report drillthrough for Power BI reports.

> [!NOTE]

> > You can't use cross-report drillthrough with individually-shared [Shared with me reports](../collaborate-share/service-share-dashboards.md#share-a-dashboard-or-report). To use cross-report drillthrough, you must access reports in the workspace from which they were a member of.

Reviewer: ```suggestion
> You can't use cross-report drillthrough with individually-shared [Shared with me reports](../collaborate-share/service-share-dashboards.md#share-a-dashboard-or-report). To use cross-report drillthrough, you must access reports in workspaces that you’re a member of.
```
